### DISCUSSION TITLE
Possible to get full format details without format_additionalfeatures?

### DISCUSSION THREAD

**Discussion by mchar7** [sbraz/pymediainfo#119] [Q&A] [answered]
I'm trying to get a little more detail from on audio track codecs and see that I would need to reference the format_additionalfeatures attribute to do all my checks.

With a DTS:X audio track, the MediaInfo CLI and GUI (most output formats) would show `"DTS XLL X"` for "format".

Is there anyway to have format return these values via pymediainfo, or is checking format_additionalfeatures the only way?

**Comment by sbraz** [comment, marked as answer]
Hi,
pymediainfo defaults to full output, similarly to `mediainfo -f`.

With full output, the "Format" attribute is repeated, e.g. from the CLI:
```
Format                                   : DTS   
Format                                   : DTS XLL X 
```

With pymediainfo, info for repeated keys can be found under `other_format`:
```
In [9]: pymediainfo.MediaInfo.parse("dtsx.mkv").audio_tracks[0].other_format
Out[9]: ['DTS XLL X']
```

You can use `to_data` to dump all attributes from your track.

If you prefer a shorter output, you can just pass `full=False` to `parse` but you'll see less info this way:
```
In [7]: pymediainfo.MediaInfo.parse("dtsx.mkv", full=False).audio_tracks[0].format
Out[7]: 'DTS XLL X'
```

**Reply by mchar7** [reply]
Thanks @sbraz,

I did not see the ```other_format``` attribute, I will reference that for my checks. I will need some info that is only provided with a full output so I can't switch to ```full=False```.

Thanks a ton for your help and maintaining pymediainfo, it is instrumental for my project.
